### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by jprovaznik**
Improved descriptions of some parameters, "parameters" section
is referenced from README as a "complete list of configurable parameters".

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -132,12 +132,24 @@ nova keypair-add --pub-key ~/.ssh/id_rsa.pub default
...
 == Deployment
 
-You can pass all environment variables to heat on command line.  However, two example environment files have been given.
-
-* ``env_origin.yaml`` is an example of the variables to deploy an OpenShift Origin 3 environment.
-* ``env_aop.yaml`` is an example of the variables to deploy an Atomic Enterprise or OpenShift Enterprise 3 environment.  Note deployment type should be *openshift-enterprise* for OpenShift or *atomic-enterprise* for Atomic Enterprise.  Also, a valid RHN subscription is required for deployment.
-
-Assuming your external network is called ``public``, your SSH key is ``default`` and your CentOS 7.2 image is ``centos72`` and your domain name is ``example.com``, this is how you deploy OpenShift Origin:
+You can pass all environment variables to heat on command line.  However,
+two example environment files have been given.
```
Comment (lebauce) [inline README.adoc]: two environment files are provided as examples

File: README.adoc [hunk 2]
```diff
@@ -132,12 +132,24 @@ nova keypair-add --pub-key ~/.ssh/id_rsa.pub default
...
+* ``env_origin.yaml`` is an example of the variables to deploy an OpenShift
+  Origin 3 environment.
+* ``env_aop.yaml`` is an example of the variables to deploy an Atomic
+  Enterprise or OpenShift Enterprise 3 environment.  Note deployment type
+  should be *openshift-enterprise* for OpenShift or *atomic-enterprise*
+  for Atomic Enterprise.  Also, a valid RHN subscription is required
+  for deployment.
+
+Here is a sample of environment file which uses a subset of parameters which
+can be set by user to configure OpenShift deployment. All configurable
```
Comment (lebauce) [inline README.adoc]: by the user

File: README.adoc [hunk 3]
```diff
@@ -132,12 +132,24 @@ nova keypair-add --pub-key ~/.ssh/id_rsa.pub default
...
+parameters including description can be found in `parameters` section in
```
Comment (lebauce) [inline README.adoc]: in the `parameters` section

File: README.adoc [hunk 4]
```diff
@@ -132,12 +132,24 @@ nova keypair-add --pub-key ~/.ssh/id_rsa.pub default
...
+the link:openshift.yaml[main template]. Assuming your external
+network is called ``public``, your SSH key is ``default`` and your
```
Comment (lebauce) [inline README.adoc]: `default`, your

File: openshift.yaml [hunk 5]
```diff
@@ -118,20 +118,30 @@ parameters:
...
   dns_nameserver:
     type: comma_delimited_list
-    description: address of a dns nameserver reachable in your environment
+    description: Addresses of a dns nameserver reachable in your environment
     default: 8.8.8.8
 
   skip_dns:
     type: boolean
+    description: >
+      If set to true DNS server is set not on bastion node for resolving
```
Comment (lebauce) [inline openshift.yaml]: is not set

**Comment by jprovaznik** [comment]
fixed, thanks

**Pull request merged**